src/ToDo.UI/Views/Dialogs/RenameListFlyout.xaml

```diff
@@ -1,50 +1,54 @@
-﻿<Page x:Class="ToDo.Views.Dialogs.RenameListFlyout"
-	  xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
-	  xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-	  xmlns:local="using:ToDo.Views.Dialogs"
-	  xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
-	  xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
-	  xmlns:uen="using:Uno.Extensions.Navigation.UI"
-	  xmlns:utu="using:Uno.Toolkit.UI"
-	  xmlns:um="using:Uno.Material"
-	  mc:Ignorable="d"
-	  Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
-
-	<utu:AutoLayout Spacing="20"
-					Padding="20,20,20,20"
-					CornerRadius="8"
-					Background="{ThemeResource MaterialSurfaceBrush}">
-
-		<TextBlock Foreground="{ThemeResource MaterialOnSurfaceBrush}"
-				   TextWrapping="Wrap"
-				   Text="Rename list"
-				   Style="{StaticResource MaterialHeadline6}" />
-
-		<TextBox x:Name="ListNameText"
-				 PlaceholderText="Rename list"
-				 Text="{Binding ListName, Mode=TwoWay}"
-				 Style="{StaticResource MaterialFilledTextBoxStyle}" />
-
-		<utu:AutoLayout Spacing="10"
-						Orientation="Horizontal">
-
-			<utu:AutoLayout Spacing="10"
-							Orientation="Horizontal"
-							utu:AutoLayout.CounterAlignment="Start"
-							utu:AutoLayout.PrimaryAlignment="Stretch" />
-
-			<Button Content="Cancel"
-					uen:Navigation.Request="-"
-					utu:AutoLayout.CounterAlignment="Start"
-					Style="{StaticResource MaterialTextButtonStyle}" />
-
-			<Button Content="Rename"
-					uen:Navigation.Request="-"
-					uen:Navigation.Data="{Binding Text, ElementName=ListNameText}"
-					utu:AutoLayout.CounterAlignment="Start"
-					Style="{StaticResource MaterialContainedButtonStyle}" />
-
-		</utu:AutoLayout>
-
-	</utu:AutoLayout>
+﻿<Page
```

Comment:
What is the real change here? Please update the xaml for only the lines you made changes and not the entire page.

Reply:
Also, please grab the XamlStyler visual-studio extension for formatter.

Reply:
I didn't change entire, using XamlStyler and save the changes was made it. The real change was add the command for renaming of the list